README.md

```diff
@@ -1 +1,10 @@
 # community
+
+![Header](/docs/_static/images/coala-header.png)
+
```

Comment:
The line contains the keyword 'coala'.

*Origin: KeywordBear, Section: `generalization`.*